packages/kit/src/core/adapt/prerender.js

```diff
@@ -217,10 +233,10 @@ export async function prerender({ cwd, out, log, config, build_data, fallback, a
 	for (const entry of config.kit.prerender.pages) {
 		if (entry === '*') {
 			for (const entry of build_data.entries) {
-				await visit(entry, null);
+				await visit(normalizeTrailingSlash(entry), null);
```

Comment:
it seems like a lot to keep track of to need to call `normalizeTrailingSlash` wherever `visit` is called. could you just call it inside `visit` as the first line of that method instead?

Reply:
Yes, that's better. Thanks. Simplified and rebased the commit.